core/domain/user_services.py

```diff
@@ -1694,3 +1747,105 @@ def is_topic_manager(user_id):
     if user_role == feconf.ROLE_ID_TOPIC_MANAGER:
         return True
     return False
+
+
+def can_review_translation_suggestions(user_id, language_code=None):
+    """Returns whether the user can review translation suggestions in any
+    language or in the given language.
+
+    NOTE: If the language_code is provided then this method will check whether
+    the user can review translation in the given language code else it will
+    check whether the user can review in any language.
```

Comment:
It's actually unclear here whether "review in any language" means "review in all languages" or "review in at least one language". Could you please clarify?

Reply:
As far as I can tell, this is not done.